containers/ddev-webserver/ddev-webserver-base-files/etc/bash.nointeractive.bashrc

```diff
@@ -1,7 +1,13 @@
 # This file is loaded in non-interactive bash shells through $BASH_ENV
-export PATH="${DDEV_COMPOSER_ROOT:-/var/www/html}/vendor/bin:${PATH}"
+case ":$PATH:" in
+    *":${DDEV_COMPOSER_ROOT:-/var/www/html}/vendor/bin:"*) ;;
+    *) PATH="${DDEV_COMPOSER_ROOT:-/var/www/html}/vendor/bin:$PATH" ;;
+esac
```

Comment:
I think this is a fine solution, but it's really, really cryptic. Please add a comment to both places explaining what it does.